hazelcast-jet-core/src/test/java/com/hazelcast/jet/SourcesTest.java

```diff
@@ -16,20 +16,60 @@
 
 package com.hazelcast.jet;
 
+import com.hazelcast.cache.jsr.JsrTestUtil;
+import com.hazelcast.client.config.ClientConfig;
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.jet.core.JobStatus;
 import com.hazelcast.jet.function.DistributedFunction;
-import org.junit.Test;
-
+import com.hazelcast.nio.Address;
+import java.io.File;
+import java.io.PrintWriter;
+import java.net.ServerSocket;
+import java.net.Socket;
 import java.util.List;
 import java.util.Map.Entry;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Ignore;
+import org.junit.Test;
 
 import static com.hazelcast.jet.Util.entry;
 import static com.hazelcast.jet.core.processor.SourceProcessors.readMapP;
+import static com.hazelcast.jet.impl.util.Util.uncheckRun;
 import static com.hazelcast.projection.Projections.singleAttribute;
 import static com.hazelcast.query.TruePredicate.truePredicate;
+import static java.nio.charset.StandardCharsets.UTF_8;
 import static java.util.stream.Collectors.toList;
 import static org.junit.Assert.assertEquals;
 
 public class SourcesTest extends PipelineTestSupport {
+    private static HazelcastInstance hz;
+    private static ClientConfig clientConfig;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        hz = Hazelcast.newHazelcastInstance();
```

Comment:
can we rename this to `remoteHz`? They can also be moved to the base class as `createRemoteCluster` as I see this code duplicated.